src/main/resources/synthetic.xml

```diff
@@ -65,5 +65,32 @@
         <property name="packageId" category="output" label="Package ID" />
     </type>
 
-	<type type="xldeploy.Server" extends="configuration.HttpConnection"/>
+    <type type="xldeploy.cliTask" extends="xlrelease.PythonScript" virtual="true" >
+        <property name="iconLocation" default="xldeploy/xldeploy-icon-black-small.png" hidden="true" />
+        <property name="taskColor" hidden="true" default="#68B749" />
+        <property name="cliHome"          category="input"/>
+        <property name="cliExecutable"    category="input" default="cli" hidden="true" />
+        <property name="xldHost"          category="input" default="DEFAULT"/>
+        <property name="xldPort"          category="input" default="DEFAULT"/>
+        <property name="xldContext"       category="input" default="DEFAULT"/>
+        <property name="xldProxyHost"     category="input" default="DEFAULT"/>
+        <property name="xldProxyPort"     category="input" default="DEFAULT"/>
+        <property name="xldSocketTimeout" category="input" default="DEFAULT"/>
+        <property name="xldUserName"      category="input" default="admin"/>
+        <property name="xldPassword"      category="input" default="admin" password="true" />
```

Comment:
Why is this not part of a global configuration type which extends from `xlrelease.Configuration`? I don't think I want to redefine for each task all these properties in my templates?